src/pytorch_lightning/lite/lite.py

```diff
@@ -268,20 +267,20 @@ def _to_lite_strategy(strategy: PLStrategy) -> LiteStrategy:
             device=strategy.root_device,
             accelerator=strategy.accelerator,
             checkpoint_io=strategy.checkpoint_io,
-            precision_plugin=_to_lite_precision_plugin(strategy.precision_plugin),
+            precision=_to_lite_precision(strategy.precision_plugin),
         )
 
     if type(strategy) is PLSingleTPUStrategy:
         return LiteSingleTPUStrategy(
             device=strategy.root_device.index,
             accelerator=strategy.accelerator,
             checkpoint_io=strategy.checkpoint_io,
-            precision_plugin=_to_lite_precision_plugin(strategy.precision_plugin),
+            precision=_to_lite_precision(strategy.precision_plugin),
         )
 
 
-def _to_lite_precision_plugin(plugin: Optional[PLPrecisionPlugin]) -> LitePrecision:
-    """Re-instantiates a PL-PrecisionPlugin as the corresponding Lite-Precision plugin."""
+def _to_lite_precision(plugin: Optional[PLPrecisionPlugin]) -> LitePrecision:
+    """Re-instantiates a PL-PrecisionPlugin as the corresponding Lite precision."""
```

Comment:
NIT: In plain text, it is still totally ok to refer to it as a plugin (see my comment above)